src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -13,26 +13,26 @@ pm_object(K::SimplicialComplex) = K.pm_simplicialcomplex
 
 
 @doc raw"""
-    SimplicialComplex(generators::Union{Vector{Vector{Int}}, Vector{Set{Int}}})
+   simplical_complex(generators::Union{Vector{Vector{Int}}, Vector{Set{Int}}})
```

Comment:
```suggestion
    simplical_complex(generators::Union{Vector{Vector{Int}}, Vector{Set{Int}}})
```
It is important that this first line is indented by (at least) four characters: that's Markdown syntax for a "code block".

Reply:
Thanks!